vslib/inc/MACsecFilter.h

```diff
@@ -0,0 +1,36 @@
+#pragma once
+
+#include "TrafficFilter.h"
+
+#include <string>
+
+namespace saivs
+{
+    class MACsecFilter : public TrafficFilter
+    {
+    public:
+        MACsecFilter(
+            _In_ const std::string &macsec_interface_name);
+
+        virtual ~MACsecFilter() = default;
+
+        virtual FilterStatus execute(
+            _Inout_ void *buffer,
+            _Inout_ ssize_t &length) override;
+
+        void enable_macsec_device(bool enable);
```

Comment:
each param needs prefix _In_ and 1 per line

Reply:
Done, please review it.